pkgs/development/python-modules/pydantic/default.nix

```diff
@@ -43,6 +43,8 @@ buildPythonPackage rec {
     export HOME=$(mktemp -d)
   '';
 
+  setupPyBuildFlags = [ "--parallel" "$NIX_BUILD_CORES" ];
```

Comment:
Is this something cython specific? You we generalize this and do a similar hook we did for pytest-xdist?

Reply:
Roger, I'll put up a new PR with these changes against `staging`.